ui/packages/catalog-ui-search/src/main/webapp/react-component/container/results-export/results-export.tsx

```diff
@@ -74,24 +76,41 @@ class ResultsExport extends React.Component<Props, State> {
     }
   }
   componentDidUpdate(_prevProps: Props, prevState: State) {
```

Comment:
```suggestion
    componentDidUpdate(_prevProps: Props, prevState: State) {
    if (
      prevState.selectedResults !== this.state.selectedResults ||
      _prevProps.transformer !== this.props.transformer
    ) {
      this.fetchExportOptions()

      this.setState({
        selectedFormat: 'Select an export option',
        downloadDisabled: true,
      })
    }
  }
  getTransformerType = () => {
    return this.props.transformer || this.state.selectedResults.length > 1
      ? 'query'
      : 'metacard'
  }
  componentDidMount() {
    this.fetchExportOptions()
  }
  fetchExportOptions = () => {
    fetch(`./internal/transformers/${this.getTransformerType()}`)
```

❗️ If we update `getTransformerType` we can simplify some of the surrounding code that relies on it.